StreamVideoTests/WebRTC/WebRTCClient_Tests.swift

```diff
@@ -628,6 +628,58 @@ final class WebRTCClient_Tests: StreamVideoTestCase {
         XCTAssert(webRTCClient.publisher?.transceiver?.sender.parameters.encodings.map(\.rid) == encodingParams.map(\.rid))
     }
     
+    func test_webRTCClient_screensharingBroadcast() async throws {
+        try await test_webRTCClient_screensharing(type: .broadcast)
+    }
+    
+    func test_webRTCClient_screensharingInApp() async throws {
+        try await test_webRTCClient_screensharing(type: .inApp)
+    }
+    
+    func test_webRTCClient_screensharing(type: ScreensharingType) async throws {
```

Comment:
This method seems more to a test utility. Maybe renaming it to `assert_` and making it private would make more sense. (Interestingly, with the `test` prefix is Xcode adding it in the TestPlans configuration)?

Reply:
yes, you are right